completions/fish/crio.fish

```diff
@@ -188,7 +191,7 @@ complete -c crio -n '__fish_seen_subcommand_from config' -f -l migrate-defaults
     Please note that the migration will overwrite any fields that have changed
     defaults between versions. To save a custom configuration change, it should
     be in a drop-in configuration file instead.
-    Possible values: "1.17"'
+    Possible values: 1.17'
```

Comment:
I'm not seeing a matchine single quote.  I only see one after  `1.17`, is one missing?  Maybe that's the hiccup?